x-pack/plugins/observability/public/components/app/section/apm/index.tsx

```diff
@@ -117,9 +117,26 @@ export function APMSection({ bucketSize }: Props) {
         <EuiFlexItem grow={false}>
           <StyledStat
             title={`${formatTpmStat(stats?.transactions.value)} tpm`}
-            description={i18n.translate('xpack.observability.overview.apm.throughput', {
-              defaultMessage: 'Throughput',
-            })}
+            description={
+              <EuiToolTip
+                content={i18n.translate('xpack.observability.overview.apm.throughputTip', {
+                  defaultMessage:
+                    'Values are calculated for transactions with type "Request" or "page-load". If neither are available, values reflect the top transaction type.',
```

Comment:
Is this accurate?

Reply:
yes correct. `page-load` is specific to RUM events (not sure if that's worth calling out)